<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPServerDialogBox</class>
 <widget class="QWidget" name="UDPServerDialogBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>736</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>736</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>736</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>849</width>
     <height>716</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="ipSelector">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findADCPushButton">
          <property name="text">
           <string>Произвести поиск АЦП</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ModuleControl">
        <property name="text">
         <string>Модуль управления</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="layoutADCDelay">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Задержки АЦП</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="delayADC_1">
        <property name="text">
         <string>0x00050004</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="delayADC_2">
        <property name="text">
         <string>0x00010003</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="delayADC_3">
        <property name="text">
         <string>0x000500B2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="delayADC_4">
        <property name="text">
         <string>0x000500B2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="delayADC_5">
        <property name="text">
         <string>0x000500B2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="CountBoards">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>1 плата</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 платы</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 платы</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="signalPeriodGroupBox">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>217</height>
           </size>
          </property>
          <property name="title">
           <string>Частота излучения, kГц</string>
          </property>
          <widget class="QWidget" name="layoutWidget1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>401</width>
             <height>211</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Частота, кГц</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Период, мкс</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QDoubleSpinBox" name="signalFrequency">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.190000000000000</double>
              </property>
              <property name="maximum">
               <double>12500.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLCDNumber" name="signalFrequencyLcd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>43</height>
               </size>
              </property>
              <property name="smallDecimalPoint">
               <bool>false</bool>
              </property>
              <property name="digitCount">
               <number>7</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Filled</enum>
              </property>
              <property name="value" stdset="0">
               <double>192.307999999999993</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLCDNumber" name="signalPeriodLcd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>43</height>
               </size>
              </property>
              <property name="smallDecimalPoint">
               <bool>false</bool>
              </property>
              <property name="digitCount">
               <number>7</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Filled</enum>
              </property>
              <property name="value" stdset="0">
               <double>192.307999999999993</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGroupBox" name="signalLengthGroupBox">
            <property name="minimumSize">
             <size>
              <width>191</width>
              <height>181</height>
             </size>
            </property>
            <property name="title">
             <string>Длина сигнала, нс</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="signalLengthVerticalLayout">
               <item>
                <widget class="QLCDNumber" name="signalLengthLcdNumber">
                 <property name="minimumSize">
                  <size>
                   <width>163</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>163</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="intValue" stdset="0">
                  <number>5120</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDial" name="signalLengthDial">
                 <property name="maximumSize">
                  <size>
                   <width>163</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="pageStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>255</number>
                 </property>
                 <property name="sliderPosition">
                  <number>255</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="notchesVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="adcDelayGroupBox">
            <property name="minimumSize">
             <size>
              <width>191</width>
              <height>181</height>
             </size>
            </property>
            <property name="title">
             <string>Задержка АЦП, м</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="adcDelayVerticalLayout">
               <item>
                <widget class="QLCDNumber" name="adcDelayLcdNumber">
                 <property name="minimumSize">
                  <size>
                   <width>163</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>163</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="smallDecimalPoint">
                  <bool>false</bool>
                 </property>
                 <property name="digitCount">
                  <number>7</number>
                 </property>
                 <property name="value" stdset="0">
                  <double>2.400000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDial" name="adcDelayDial">
                 <property name="maximumSize">
                  <size>
                   <width>163</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="pageStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="notchesVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="workModeGroupBox">
          <property name="title">
           <string>Режим работы</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="workModeVerticalLayout">
             <item>
              <widget class="QRadioButton" name="workMode0RadioButton">
               <property name="text">
                <string>В реальном времени без обработки</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="workMode2RadioButton">
               <property name="text">
                <string>Запись в буферную память</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelAccumMode">
             <property name="text">
              <string>Режим накопления</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="accumMode">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>128</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>256</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>512</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1024</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="sdramModeGroupBox">
            <property name="title">
             <string>режим буферирования</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="sdramModeVerticalLayout">
               <item>
                <widget class="QRadioButton" name="strobRadioButton">
                 <property name="text">
                  <string>запись стробами</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="continuousRadioButton">
                 <property name="text">
                  <string>неприрывная запись</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="clkGroupBox">
            <property name="title">
             <string>Тактовая частота</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <layout class="QVBoxLayout" name="clkVerticalLayout">
               <item>
                <widget class="QRadioButton" name="innerClkRadioButton">
                 <property name="text">
                  <string>Внутренняя</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="outerClkRadioButton">
                 <property name="text">
                  <string>Внешняя</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="azimuthGroupBox">
          <property name="title">
           <string>Сигналы азимута</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="azimuthVerticalLayout">
             <item>
              <widget class="QRadioButton" name="outerAzimuthRadioButton">
               <property name="text">
                <string>Формирование азимута от внешнего источника</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="innerAzimuthRadioButton">
               <property name="text">
                <string>Формирование азимута от внутреннего источника</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="countAzimut">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>64</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="otherVerticalLayout">
          <item>
           <widget class="QCheckBox" name="nsModeCheckBox">
            <property name="text">
             <string>Режим 1нс вкл/выкл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lvdsCheckBox">
            <property name="text">
             <string>Разрешить/запретить формирование LVDS</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="GHzCheckBox">
            <property name="text">
             <string>Разрешить/запретить формирование сигнала 1ГГц</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="testDataCheckBox">
            <property name="text">
             <string>Режим тестирования вкл/выкл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoTrend">
            <property name="text">
             <string>Авто тренд</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="adcNumberHorizontalLayout">
          <item>
           <widget class="QLabel" name="adcNumberLabel">
            <property name="text">
             <string>Количество отсчетов АЦП</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="adcNumberComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>1024</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2048</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3072</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4096</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5120</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6144</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7168</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8192</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>innerAzimuthRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>countAzimut</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
